requirements.txt

```diff
@@ -1,8 +1,10 @@
+six
```

Comment:
I think it has to be used in `setup.tmpl` as well.

Reply:
Done!